lib/_stream_legacy.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
```

Comment:
The reason is? IMO the legacy stream class should not be internal class, it's exposed to user-land, right?